### PR TITLE
feat: update codebase

### PR DESCRIPTION
**Pull request by johnny243**
- updates to `sn-editor-kit@1.0.9`, which supports the new `component-relay` package

### CODE REVIEW

File: app/main.js [hunk 1]
```diff
@@ -4,5 +4,5 @@ import App from './App';
 
 ReactDOM.render(
   <App />,
-  document.body
+  document.body.appendChild(document.createElement('div'))
```
Comment (mobitar) [inline app/main.js:7]: Why is this needed?

Comment (johnny243) [reply app/main.js:7]: On latests React versions, rendering into the `body` directly throws the following error:

`Warning: render(): Rendering components directly into document.body is discouraged, since its children are often manipulated by third-party scripts and browser extensions. This may lead to subtle reconciliation issues. Try rendering into a container element created for your app.`

**Review by mobitar — APPROVED** [review]
